provider/dynamodb.go

```diff
@@ -380,26 +428,30 @@ func (store *dynamodbOnlineStore) GetImport(id ImportID) (Import, error) {
 	if output.ImportTableDescription.FailureCode != nil {
 		errorMessage = *output.ImportTableDescription.FailureCode
 	}
-	return S3Import{id: id, status: *output.ImportTableDescription.ImportStatus, errorMessage: errorMessage}, nil
+	return S3Import{id: id, status: string(output.ImportTableDescription.ImportStatus), errorMessage: errorMessage}, nil
 }
 
 func (table dynamodbOnlineTable) Set(entity string, value interface{}) error {
+	dynamoValue, err := serializers[table.version].Serialize(table.valueType, value)
```

Comment:
minor (no action necessary). what do you think of `table.Seriallizer.Serialize(...`?

Reply:
I don't understand your fancy words

Reply:
current code makes me think the table.version can change between set calls

Reply:
serde is common naming for **ser**ializer**de**serializer stuff